src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIQueueWebsocketService.java

```diff
@@ -70,22 +70,22 @@ public void addListeners() {
         this.queue.addItemListener(new QueuedBuildJobItemListener(), true);
         this.processingJobs.addLocalEntryListener(new ProcessingBuildJobItemListener());
         this.buildAgentInformation.addLocalEntryListener(new BuildAgentListener());
-        // localCIBuildQueueWebsocketService will be autowired only if scheduling is active
-        Objects.requireNonNull(localCIBuildQueueWebsocketService, "localCIBuildQueueWebsocketService must be non-null when scheduling is active.");
+        // localCIWebsocketMessagingService will be autowired only if scheduling is active
+        Objects.requireNonNull(localCIWebsocketMessagingService, "localCIWebsocketMessagingService must be non-null when scheduling is active.");
```

Comment:
This could also be done with an Optional<LocalCIWebsocketMessagingService> which I think we use quite often in the codebase.

Reply:
This was still from before having refactored this class. I will remove it as it's unnecessary.